VMEncryption/test/test_resource_disk_util.py

```diff
@@ -211,3 +211,24 @@ def test_automount(self, mock_try_remount, mock_encrypt_format_mount, mock_is_pl
         mock_encrypt_format_mount.return_value = False
         self.assertEqual(self.resource_disk.automount(), False)
         mock_encrypt_format_mount.assert_called_once()
+
+    @mock.patch('main.ResourceDiskUtil.ResourceDiskUtil._is_crypt_mounted')
+    @mock.patch('main.ResourceDiskUtil.ResourceDiskUtil.encrypt_format_mount')
+    def test_encrypt_resource_disk(self, mock_efm, mock_icm):
```

Comment:
Just a heads up these tests may need to be updated in the future so they can run independently without reliance on presence of disks, folders, or current user privileges.  For example if they result in a call to "mount" which requires root privileges.

Reply:
Thanks for the heads up Eric. This particular test does not run any privileged command. I just mocked return values of two functions.